vcpkg build failed [undefined reference to `vcpkg::VcpkgCmdArguments::OVERLAY_TRIPLETS_ENV']

**Describe the bug**

Failed to build vcpkg.

**Environment**
- OS: Debian 9.13
- Compiler: g++ 6.3.0

**To Reproduce**

$ git clone https://github.com/microsoft/vcpkg
$ ./vcpkg/bootstrap-vcpkg.sh
-- The CXX compiler identification is GNU 6.3.0
-- Check for working CXX compiler: /usr/bin/g++-6
-- Check for working CXX compiler: /usr/bin/g++-6 - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting the C++ compiler in use
-- Detecting the C++ compiler in use - gcc
-- Detecting the C++ standard library
-- Detecting the C++ standard library - libstdc++
-- Detecting how to use the C++ filesystem library
-- Detecting how to use the C++ filesystem library - <experimental/filesystem> with -lstdc++fs
-- Looking for C++ include pthread.h
-- Looking for C++ include pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE  
-- Configuring done
-- Generating done
-- Build files have been written to: /home/user/vcpkg/toolsrc/build.rel
[0/2] Re-checking globbed directories...
[83/83] Linking CXX executable vcpkg
FAILED: vcpkg 
: && /usr/bin/g++-6  -O3 -DNDEBUG   CMakeFiles/vcpkg.dir/src/vcpkg.cpp.o  -o vcpkg  libvcpkglib.a  -pthread  -lstdc++fs && :
libvcpkglib.a(vcpkgcmdarguments.cpp.o): In function `vcpkg::VcpkgCmdArguments::imbue_from_environment()':
vcpkgcmdarguments.cpp:(.text+0x4bf7): undefined reference to `vcpkg::VcpkgCmdArguments::OVERLAY_TRIPLETS_ENV'
vcpkgcmdarguments.cpp:(.text+0x4bfe): undefined reference to `vcpkg::VcpkgCmdArguments::OVERLAY_TRIPLETS_ENV'
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.

--- a/toolsrc/src/vcpkg/vcpkgcmdarguments.cpp
+++ b/toolsrc/src/vcpkg/vcpkgcmdarguments.cpp
@@ -911,6 +911,7 @@ namespace vcpkg
     constexpr StringLiteral VcpkgCmdArguments::TRIPLET_ARG;
     constexpr StringLiteral VcpkgCmdArguments::OVERLAY_PORTS_ENV;
     constexpr StringLiteral VcpkgCmdArguments::OVERLAY_PORTS_ARG;
+    constexpr StringLiteral VcpkgCmdArguments::OVERLAY_TRIPLETS_ENV;
     constexpr StringLiteral VcpkgCmdArguments::OVERLAY_TRIPLETS_ARG;
 
     constexpr StringLiteral VcpkgCmdArguments::BINARY_SOURCES_ARG;

